tests/test_global.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env pytest
+import torch
+
+import torchdynamo.testing
+from torchdynamo.testing import same
+
+from . import test_global_declaration
+
+torchdynamo.config.debug = True
```

Comment:
If you subclass `torchdynamo.testing.TestCase` this should get turned on automatically.